Goals.md

```diff
@@ -14,3 +14,4 @@
 - [x] Own a boat
 - [x] Be not famous
```

Comment:
you're not making enough changes!!! hot code review me please

Reply:
wowzers @rizel-botany 🙉 🎧 ❤️